### PR TITLE
init: container, bump to 4.6.0

### PR DESCRIPTION
**Pull request by kitplummer**

### CODE REVIEW

File: .github/workflows/container.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
         with:
           node-version: "20.x"
           cache: 'npm'
-      - run: npm ci
+      - run: npm install
```
Comment (ajfabbri) [inline .github/workflows/container.yml:37]: Curious why this is needed. IIRC `ci` is the better option. Should we fix the issue, or add a  `TODO` comment here explaining what needs to be done to fix clean install?

Comment (kitplummer) [reply .github/workflows/container.yml:37]: The only _real_ thing `ci` provides is a hard failure if the package and package-lock aren't in sync.  I think I was working around a mismatch that shouldn't have existed.  Going to switch back to `ci`.

File: .github/workflows/container.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: Create and publish a container
+
+# Configures this workflow to run every time a change is pushed to the branch called `release`.
```
[the remark this answers is not in the document]
Comment (kitplummer) [reply .github/workflows/container.yml:3]: Will remove the comments from the GH template when I switch this to go on `main`.

**Comment by kitplummer** [comment]
Screenshot 2024-03-18 at 5 38 34 PM

Now appending the commit hash, as a first step to "version" the container on build/deploy.

File: .github/workflows/container.yml (hunk 1 above)
Comment (kitplummer) [reply .github/workflows/container.yml]: Fix it, was an issue with the mismatch.

File: start_node.sh [hunk 3]
```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
+# trap ctrl-c and call ctrl_c()
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "** Trapped CTRL-C"
+}
+
```
[the remark this answers is not in the document]
Comment (kitplummer) [reply start_node.sh:2]: Removed this altogether.  Was trying to work around tty issue with not using `-it`

**Review by ajfabbri — APPROVED** [review]
LGTM. I confirmed I can pull the image you shared. Let me know if you need help with other testing.

**Comment by kitplummer** [comment]
> LGTM. I confirmed I can pull the image you shared. Let me know if you need help with other testing.

Will update the README to include a section on running from the container.  I need to try and test it locally myself - with the port forwards.  :D

**Comment by kitplummer** [comment]
This thing is all kinds of outawhack.

**Pull request closed without merging**